AsyncNetworkService/Common/RequestLogger.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  RequestLogger.swift
+//  
+//
+//  Created by Paul Alvarez on 17/10/23.
+//
+
+import Foundation
+
+public final class RequestLogger {
+    public static let shared = RequestLogger()
+
+    public var logs: [RequestLog] = []
+
+    func log(request: RequestLog) {
+        logs.append(request)
```

Comment:
got crash on launch here 
![image](https://github.com/dustid/AsyncHTTPNetworkService/assets/127774758/c7e62da6-c83b-4626-bae8-de34774d3f5d)

Reply:
Fixed! Review it again please